### PR TITLE
Amls 5765 - Withdraw and Deregister reasons content update

### PR DESCRIPTION
**Pull request by matthewharris1**
Content changes, H1 changes, URL changes for Deregister and Withdrawal reasons pages.

## How Has This Been Tested?
Unit
Manual

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -1306,22 +1310,24 @@ dateofchange.hint = For example, 31 03 1980.
...
-deregistration.reason.lbl.03 = Under another supervisor
-deregistration.reason.lbl.04 = Change of Legal Entity
-deregistration.reason.lbl.05 = HVD - policy of not accepting high value cash payments
-deregistration.reason.lbl.06 = Other
-deregistration.reason.lbl.specify.other.reason = Please specify
+deregistration.reason.lbl.01 = Business is out of scope as no longer carrying out activities covered by the Money Laundering Regulations
+deregistration.reason.lbl.02 = Business is not trading in its own right because it is now an agent of a money service business
+deregistration.reason.lbl.03 = Business is registered with another supervisory body
+deregistration.reason.lbl.04 = Legal entity of your business has changed
+deregistration.reason.lbl.05 = You're registered as a high value dealer but no longer make or accept high value cash payments
```
Comment (slavvco) [inline conf/messages]: wrong '

File: conf/app.routes [hunk 2]
```diff
@@ -50,8 +50,8 @@ POST        /your-registration/withdraw-application-reason
 GET         /your-registration/de-register-application                                @controllers.deregister.DeRegisterApplicationController.get()
 POST        /your-registration/de-register-application                                @controllers.deregister.DeRegisterApplicationController.post()
 
-GET         /your-registration/de-register-application-reason                         @controllers.deregister.DeregistrationReasonController.get()
-POST        /your-registration/de-register-application-reason                         @controllers.deregister.DeregistrationReasonController.post()
+GET         /your-registration/de-register-reason                                     @controllers.deregister.DeregistrationReasonController.get()
```
Comment (slavvco) [inline conf/app.routes]: url is not correct

File: conf/messages (hunk 1 above)
Comment (matthewharris1) [reply conf/messages]: Soo close! Should be sorted now

**Comment by slavvco** [comment]
test this please

**Pull request merged**